test/OpenTelemetry.Instrumentation.Wcf.Tests/TelemetryBindingElementForHttpTests.cs

```diff
@@ -35,47 +35,48 @@ public class TelemetryBindingElementForHttpTests : IDisposable
 {
     private readonly Uri serviceBaseUri;
     private readonly HttpListener listener;
-    private readonly EventWaitHandle initializationHandle;
```

Comment:
curious - why change `initializationHandle` from a member variable to be a local variable?

Reply:
there was no need for it to be a member variable - it's only used as a local variable within that method

Reply:
Gotcha! I originally thought it was something related to nullability in this context. thanks!